src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,24 @@
+@import "../../patternfly-utilities";
+
 .pf-l-gallery {
-  display: grid;
-  grid-gap: 1rem; //revisit this value
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
-  padding: 0;
-  margin: 0;
+  $pf-l-gallery--GridGap: 1rem !default;
```

Comment:
Will this be tie to a gutter global variable when we define it?

Reply:
Yeah. It might change names depending on how we implement the gutters though. I just left it generic for now, since we still need to discuss the gutters a little further.